rust_programs/ttf_renderer/src/glyphs.rs

```diff
@@ -562,3 +573,107 @@ fn interpret_values_via_flags(
     }
     values
 }
+
+fn bezier_2d(start: isize, ctrl: isize, end: isize, t: f64) -> isize {
+    return ((start as f64) + (2 as f64)*t*(ctrl as f64) + t*t*((end - ctrl) as f64)).round() as isize;
+}
+
+fn points_for_polygon(
+    start_index: usize,
+    end_index: usize,
+    points: &Vec<isize>,
+    flags: &Vec<Vec<GlyphOutlineFlag>>,
+    interpolate: bool
+) -> Vec<isize> {
+    
+    let mut values = vec![];
+
+    if !interpolate {
+        return points[start_index..=end_index].to_vec().clone();
```

Comment:
I think `to_vec` already clones the slice, so you can drop the second `clone` here.

Reply:
Done